### ISSUE TITLE
Android: Cannot cancel/retry/run tests/etc.

### ISSUE THREAD

**Issue by qui-jason** [jorgefspereira/persona_flutter#51]
PR #45 seems incorrect to me - the conditional `isResultSubmitted` prevents subsequent attempts on Android if the user clicks cancel, errors out. This also presents a problem in automated testing, where the framework is not necessarily reloaded across runs that need to successfully complete in dev. To be clear, this change made it such that callbacks after inquiry fire only once, meaning that only two inquiry attempts can occur and the second will leave the UI hanging.

I suggest reverting #45, perhaps there is another issue at play in the submitter's code. We don't suffer from any issues when clicking cancel. Alternatively, provide a way to reset this variable, or do it when `start()` is called.

**Comment by qui-jason** [comment]
It could go [here](https://github.com/jorgefspereira/persona_flutter/blob/master/android/src/main/kotlin/com/jorgefspereira/persona_flutter/PersonaFlutterPlugin.kt#L158) for example, something like:

```kotlin
isResultSubmitted = false
```
